### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by berry120**
As per #171 - fix some typos in the codebase without breaking the public API. (Typos fixed in a combination of private fields & comments.)

### CODE REVIEW

File: src/main/java/mdlaf/themes/AbstractMaterialTheme.java [hunk 1]
```diff
@@ -1090,7 +1090,7 @@ public boolean getEnableArrowScrollBar() {
     return false;
   }
 
-  // get jslider
+  // get JSlider
```
Comment (vincenzopalazzo) [inline src/main/java/mdlaf/themes/AbstractMaterialTheme.java:1093]: This comment can be removed. It have no sense here

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]
LGTM, I have only a small comment but nothings else.

Thanks.

File: src/main/java/mdlaf/themes/AbstractMaterialTheme.java (hunk 1 above)
Comment (berry120) [reply src/main/java/mdlaf/themes/AbstractMaterialTheme.java]: Thanks, that's now removed!

**Review by vincenzopalazzo — APPROVED** [review]
LGTM, thanks for the amazing work 😸 

@vincenzopalazzo approve this PR.

**Pull request merged**